src/isar/services/utilities/scheduling_utilities.py

```diff
@@ -24,28 +30,192 @@ class SchedulingUtilities:
-        )
+        """Start mission
+
+        Raises
+        ------
+        HTTTPException 408 Request timeout
+            If there is a timeout while communicating with the state machine
+        """
+        try:
+            self._send_command(
+                StartMissionMessage(
+                    mission=deepcopy(mission), initial_pose=initial_pose
+                ),
+                self.queues.start_mission,
+            )
+        except QueueTimeoutError:
+            error_message = "Timeout - Failed to start mission in ISAR"
+            self.logger.error(error_message)
+            raise HTTPException(
+                status_code=HTTPStatus.REQUEST_TIMEOUT, detail=error_message
+            )
+        self.logger.info("OK - Mission started in ISAR")
 
     def pause_mission(self) -> ControlMissionResponse:
-        return self._send_command(True, self.queues.pause_mission)
+        """Pause mission
+
+        Raises
+        ------
+        HTTTPException 408 Request timeout
+            If there is a timeout while communicating with the state machine
+        """
+        try:
+            return self._send_command(True, self.queues.pause_mission)
+        except QueueTimeoutError:
+            error_message = "Timeout - Failed to pause mission"
+            self.logger.error(error_message)
+            raise HTTPException(
+                status_code=HTTPStatus.REQUEST_TIMEOUT, detail=error_message
+            )
+        finally:
+            self.logger.info("OK - Mission successfully paused")
 
     def resume_mission(self) -> ControlMissionResponse:
-        return self._send_command(True, self.queues.resume_mission)
+        """Resume mission
+
+        Raises
+        ------
+        HTTTPException 408 Request timeout
+            If there is a timeout while communicating with the state machine
+        """
+        try:
+            return self._send_command(True, self.queues.resume_mission)
+        except QueueTimeoutError:
+            error_message = "Timeout - Failed to resume mission"
+            self.logger.error(error_message)
+            raise HTTPException(
+                status_code=HTTPStatus.REQUEST_TIMEOUT, detail=error_message
+            )
+        finally:
+            self.logger.info("OK - Mission successfully resumed")
 
     def stop_mission(self) -> ControlMissionResponse:
-        return self._send_command(True, self.queues.stop_mission)
+        """Stop mission
+
+        Raises
+        ------
+        HTTTPException 408 Request timeout
+            If there is a timeout while communicating with the state machine
+        """
+        try:
+            stop_mission_response: ControlMissionResponse = self._send_command(
+                True, self.queues.stop_mission
+            )
+        except QueueTimeoutError:
+            error_message = "Timeout - Failed to stop mission"
+            self.logger.error(error_message)
+            raise HTTPException(
```

Comment:
408 Request timeout implies a user error (all 400 codes do), I believe this should be a 500 internal server error as it has nothing to do with the user input

Reply:
Good catch, changed to 500 Internal server error